data/blog/electron-4-0.md

```diff
@@ -0,0 +1,71 @@
+
+---
+
+# What's New?
+
+A large part of Electron's functionality is provided by Chromium, Node.js, and V8, the core components that make up Electron. As such, a key goal for the Electron team is to keep up with changes to these projects as much as possible, providing developers who build Electron apps access to new web and JavaScript features. To this end, Electron 4 features major version bumps to each of these components; Electron v4.0.0 includes Chromium `x.x.x`, Node `x.x.x`, and V8 `x.x.x`.
+
+In addition, Electron 4 includes changes to Electron-specific APIs. You can find a summary of the major changes in Electron 4 below; for the full list of changes, check out the [Electron v4.0.0 release notes](TODO_RELEASE_NOTES).
+
+## Breaking Changes
+
+### Single Instance Locking
+
+Previously, to make your app a Single Instance Application (ensuring that only one instance of your app is running at any given time), you could use the `app.makeSingleInstance()` method. Starting in Electron 4.0, you must use `app.requestSingleInstanceLock()` instead. The return value of this method indicates whether or not this instance of your application successfully obtained the lock. If it failed to obtain the lock, you can assume that another instance of your application is already running with the lock and exit immediately.
+
+For an example of using `requestSingleInstanceLock()` and information on nuanced behavior on various platforms, [see the documentation for `app.requestSingleInstanceLock()` and related methods](https://electronjs.org/docs/api/app#apprequestsingleinstancelock) and [the `second-instance` event](https://electronjs.org/docs/api/app#event-second-instance).
+
+### `win_delay_load_hook`
+
+When building native modules for windows, the `win_delay_load_hook` variable in the module's `binding.gyp` must be true (which is the default). If this hook is not present, then the native module will fail to load on Windows, with an error message like `Cannot find module`. [See the native module guide](https://electronjs.org/docs/tutorial/using-native-node-modules#a-note-about-win_delay_load_hook) for more information.
+
+### Accessing `electron.screen` in Renderer Processes
+
+Instead of accessing the [screen module](https://electronjs.org/docs/api/screen) in a renderer process via `require('electron').screen`, you should use `require('electron').remote.screen`.
+
+### Accessing `require` in Sandboxed Renderer Processes
+
+If your renderer process is sandboxed, instead of requiring modules with `require('module')`, you should use `require('electron').remote.require('module')`.
```

Comment:
Absolutely; this is a bit of a placeholder because I'm not sure about the specifics of the change and the breaking changes doc has no specific details.